Cartfile

```diff
@@ -2,5 +2,5 @@ github "Alamofire/Alamofire" == 3.1.2
 github "DaveWoodCom/XCGLogger" == 3.1.1
 github "Hearst-DD/ObjectMapper" == 1.0.1
 github "rhodgkins/SwiftHTTPStatusCodes" == 2.0.1
-github "daltoniam/Starscream" == 1.0.2
-github "tristanhimmelman/AlamofireObjectMapper" == 2.1.0
\ No newline at end of file
+github "daltoniam/Starscream" == 1.1.1
```

Comment:
Yes. 
Every time you update a dependency you need to get the approval, it think it's pretty easy once you have it approved. @mbouldin can probably help with this